core-bundle/src/Resources/contao/dca/tl_search_index.php

```diff
@@ -19,7 +19,8 @@
 			(
 				'id' => 'primary',
 				'pid' => 'index',
-				'word' => 'index'
+				'termId' => 'index',
+				'termId,pid' => 'unique'
```

Comment:
If we have a composite index, we do no longer need to index the columns separately, do we?

Reply:
Yes, I think this is true for the `termId` index as it is a prefix for the `termId,pid` prefix. But the `pid` index needs to stay IMO because searching by `pid` only cannot make use uf the `termId,pid` index.

Reply:
`termId` index got removed in dbc587bd1ad1e2d6650943bcfce868927e327c64

Reply:
This statement is correct, imho :)